python/social_bot/envs/grocery_ground.py

```diff
@@ -75,7 +75,7 @@ def __init__(self,
                  fail_distance_thresh=3,
                  random_range=10.0,
                  random_goal=False,
-                 sparse_reward=False,
+                 sparse_reward=True,
```

Comment:
Do you also need to change the gin files? Because probably they are for sparse_reward=False?

Reply:
Yes, we should. For some experiment does not care sparse reward or not in the future, this option can speed up the training progress.